### PR TITLE
Fix issue with svgs not loading in widget

### LINKED ISSUE #6: Svg loading errors on "Connecting to ..." screen
image

### PR DESCRIPTION
**Pull request by tphalp**
Fixes #6 

With fix:
image

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,111 +1,36 @@
 {
```
Comment (tphalp) [inline tsconfig.json:1]: This just removes the unused params from the tsconfg file....no actual changes, except for removing comments.

File: rollup.config.mjs [hunk 2]
```diff
@@ -62,6 +61,7 @@ export default [
         dedupe: ['d3', 'util', 'lodash', 'react', 'react-dom'], // <-- This could be improved...not sure if it is needed
       }),
       nodePolyfills(),
+      image(),
```
Comment (tphalp) [inline rollup.config.mjs:64]: The `Image` plugin is what ended up getting this to work.

File: package.json [hunk 3]
```diff
@@ -96,7 +96,7 @@
     "@babel/preset-react": "^7.23.3",
     "@rollup/plugin-babel": "^6.0.4",
     "@rollup/plugin-commonjs": "^25.0.7",
-    "@rollup/plugin-inject": "^5.0.5",
```
Comment (tphalp) [inline package.json:99]: This was left over from some attempts to fix a prior issue. Removing since it's not actually used.

File: package.json [hunk 4]
```diff
@@ -107,7 +107,6 @@
     "rollup": "^4.12.0",
     "rollup-plugin-dts": "^6.1.0",
     "rollup-plugin-polyfill-node": "^0.13.0",
-    "rollup-plugin-svg-import": "^3.0.0",
```
Comment (tphalp) [inline package.json:110]: Remove old plugin that didn't work as expected.

**Review by Craiting — APPROVED** [review]
Nice!

**Pull request merged**